flask_fas_openid.py

```diff
@@ -89,7 +89,18 @@ def __init__(self, app=None):
         self.postlogin_func = None
         self.app = app
         if self.app is not None:
-            self._init_app(app)
+            self.init_app(app)
+
+    def init_app(self, app):
+        """ Constructor for the Flask application. """
+        self.app = app
```

Comment:
It looks like this happens on line 90 above, which I think means you can also drop the app argument from this method.

Reply:
The ``__init__`` method does not require ``app`` to be specified (defaulting to ``None``) while ``init_app`` does require ``app`` to be specified, so if ``__init__`` is called without ``app``, which is the case when you manually call ``init_app``, then ``self.app`` needs to be set in ``init_app`` :)